src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/Licence.kt

```diff
@@ -247,4 +254,36 @@ data class Licence(
 
   @Schema(description = "If ARD||CRD falls on Friday/Bank holiday/Weekend then it is eligible for early release)")
   val isEligibleForEarlyRelease: Boolean = false,
-)
+) {
+  fun transformToPublicLicence(): uk.gov.justice.digital.hmpps.createandvaryalicenceapi.resource.publicApi.model.licence.Licence {
```

Comment:
This needs to be in the correct file - all the public API transforms live here 
: `uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/publicApi/ToModelTransformers.kt`

Reply:
ideally we wouldn't need to mock the transformer functions, they're just pure functions.

Reply:
yup, but these were not under test and were a dependency :)

Reply:
I'm not sure I agree that they are a dependency as such.. in kotlin they are resolved to static invocations. 
They're just pure functions with no side effects and there's not really any harm calling them directly in tests.
   
I think the reason why it mocking didn't work is that top level functions are static methods and need to mocked like [this](https://notwoods.github.io/mockk-guidebook/docs/mocking/extension/#:~:text=Kotlin%20lets%20you%20write%20functions,can%20be%20mocked%20using%20mockkStatic%20.)

Reply:
fixed by using the actual licence object

Reply:
are we able to get rid of this now? can you just use the version in transformers?